src/makeRelativeDate.ts

```diff
@@ -0,0 +1,164 @@
+type RelativeDateOptions = {
+	notAfter?: number;
+	format?: FormatType;
+	showTime?: boolean;
+};
+
+type FormatType = 'short' | 'med' | 'long';
```

Comment:
```suggestion
type Format = 'short' | 'med' | 'long';
```

Reply:
That would be a namespace clash with `Format` which is an existing type used in all the platforms. You could get away with it as it's scoped but it might cause issues with global find and replace actions (which we've been doing a lot of recently!)

Reply:
oooof wow so `Format` is now a reserved word? surely that's a massive footgun awaiting non-experts? (but not one for this PR)